votes/0179-move-match-template.md

```diff
@@ -0,0 +1,18 @@
+# A Vote for moving match_template to tikv org
+
+## Proposal
+
+[match_template](https://github.com/tikv/tikv/tree/master/components/match_template) is a procedural macro that generates repeated match arms by pattern, which is almost the only way to dispatch code path to static types depending on a runtime value. Since it's general enough, I propose to move it to TiKV org and then publish it to crates.io so that the rust community could be benifit from it.
+
+## Deadline
+
+The vote will be open for at least 6 days unless there is an objection or not enough votes.
+
+## Result
+
+Approved by 3 binding votes.    
+
+- busyjay (binding)
+- skyzh (binding)
+- andylokandy (binding)
```

Comment:
Interesting, so author can approve their own proposal?

Reply:
The GOVERNANCE writes "Binding Voters" in default team decisions are "Active maintainers", no matter whether the maintainer is the proposer.

If you're a maintainer, you should be able to vote for your proposal :)

The exceptions among the GOVERNANCE file are about revoking a role permission, said:

> except the reviewer in question if they are also a maintainer
> except the committer in question if they are also a maintainer
> except the member in question if they are also a committee member

Since related votes are in form of Consensus, these exceptions avoid the person in question always responses binding veto so that revoke permission can never perform.